src/main/resources/messages.properties

```diff
@@ -21,3 +22,29 @@ match.page=Match
 notification.message.declined={0} has declined the request for the match on {1}.
 notification.matchRequest.oneOpponent=Please verify that {0} won the match on {1}.
 notification.matchRequest.twoOpponents=Please verify that {0} and {1} won the match on {2}.
+
+sites.matchrequest=Enter match
+sites.ranking=Ranking
+sites.profile=My profile
+sites.logout=Logout
+
+statistics.rank=Rank
+statistics.points=Points
+statistics.games=Games
+statistics.victories=Victories
+statistics.defeats=Defeats
+statistics.victoryquote=Victories/Defeats
+
+matchresult.whoplayed=Who played?
+matchresult.whowon=Who won?
+matchresult.whendidyouplay=When did you play?
+matchresult.team1=Team 1
+matchresult.team2=Team 2
+matchresult.player1=Player 1
+matchresult.player2=Player 2
+matchresult.creatematch=Record Match
+matchresult.annotation=Es k�nnen entweder 2 Spieler oder jeweils ein Spieler in die Teams eingetragen werden, um ein g�ltiges Match zu erstellen.
```

Comment:
Please translate

Reply:
Also please use camelCase in the message names like: `createMatch` instead of `creatematch`.